config/defaults.conf

```diff
@@ -400,6 +400,11 @@ qqp-alt_pair_attn = 1
 qqp-alt_val_interval = 1000
 qqp-alt_lr = 0.0003
 
+chunk_size = 10 //chunk size for the ON-LSTM encoder
```

Comment:
General theme: Make sure you understand the structure and style of the file before changing it. Ask for help when you need it.
- Prepend these names with onlstm_ or something similar.
- Move this to an ON-LSTM section that's near the parameters for the other specific model architecture.
- Match formatting/style/spacing to the rest of this file.
- Add a comment explaining every single parameter.

Reply:
Done. Added a onlstm_ prefix in defaults.conf.